be/src/runtime/vectorized/statistic_result_writer.cpp

```diff
@@ -84,12 +88,39 @@ Status StatisticResultWriter::append_chunk(vectorized::Chunk* chunk) {
     return status;
 }
 
+void StatisticResultWriter::_fill_statistic_data_v2(int version, const vectorized::Columns& columns,
+                                                    const vectorized::Chunk* chunk, TFetchDataResult* result) {
+    SCOPED_TIMER(_serialize_timer);
+    DCHECK(columns.size() == 3);
+    auto versioncolumn = ColumnHelper::cast_to_raw<TYPE_BIGINT>(columns[1]);
+    auto dictColumnViewer = ColumnViewer<TYPE_VARCHAR>(columns[2]);
+
+    std::vector<TStatisticData> data_list;
+    int num_rows = chunk->num_rows();
+    data_list.resize(num_rows);
+
+    for (int i = 0; i < num_rows; ++i) {
+        data_list[i].__set_meta_version(versioncolumn->get_data()[i]);
+        if (!dictColumnViewer.is_null(i)) {
```

Comment:
implement deserilize TGlobalDict reference 
be/test/exprs/agg/aggregate_test.cpp: test_dict_merge